src/foam/nanos/cron/Schedulable.js

```diff
@@ -29,7 +29,55 @@
     'foam.nanos.auth.AuthorizationException',
     'foam.nanos.auth.AuthService',
     'foam.nanos.auth.CreatedByAware',
-    'foam.nanos.auth.Subject'
+    'foam.nanos.auth.Subject',
+
+    'java.util.Date'
+  ],
+
+  imports: [
+    'addCommas'
+  ],
+
+  requires: [
+    'foam.nanos.auth.LifecycleState',
```

Comment:
Why does Schedulable need lifecycleState, the model is not lifecycleAware.

Reply:
It uses it below.